src/DependencyInjection/NeustaConverterExtension.php

```diff
@@ -40,7 +40,7 @@ public function loadInternal(array $mergedConfig, ContainerBuilder $container):
     /**
      * @param array<string, mixed> $config
      */
-    private function registerConverterConfiguration(string $id, array $config, ContainerBuilder $container): void
+    private function registerConverterConfiguration(string $id, array $config, ContainerBuilder $container, ?string $converterSuffix): void
```

Comment:
I guess this is not nullable when there's a default value defined in the config.